examples/example_precompiles_vm/example_precompiles_vm.cpp

```diff
@@ -117,9 +99,9 @@ evmc_vm* evmc_create_example_precompiles_vm()
         EVMC_ABI_VERSION,
         "example_precompiles_vm",
         PROJECT_VERSION,
-        [](evmc_vm*) {},
+        [](evmc_vm* /*vm*/) {},
```

Comment:
these and the ones with `evmc_result` are kind of tautological... not sure if better just to allow no comment

Reply:
I agree here.